adobe/spectrum-web-components
# fix(picker): use "modal" as the menu overlay interaction

## Description
Use "modal" as the overlay type of Picker menus so that the page won't scroll while they are open.

## Related issue(s)
fixes #1424
refs #697

## Motivation and context
- Menus following their triggers when they go out of view is strange.
- If you can't scroll, you might not be able to touch scroll, and the page might not get larger.

## How has this been tested?

    1. Visit https://westbrook-picker-modal--spectrum-web-components.netlify.app/
    2. Scroll to page down and up to confirm you can
    3. Open one of the theme customization Pickers in the top right
    4. Attempt to scroll the page down and up to confirm you can not

## Types of changes

## Checklist
